src/main/java/uk/gov/hmcts/reform/pcqbackend/model/PcqAnswerRequest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.hmcts.reform.pcqbackend.model;
+
+import java.io.Serializable;
+
+/**
+ * This is the object representing the REST request for the SubmitAnswers API.
+ */
+public class PcqAnswerRequest implements Serializable {
+
+    public static final long serialVersionUID = 4328743L;
+
+    private int pcqId;
+
+    private String caseId;
+
+    private String partyId;
+
+    private int channel;
+
+    private String completedDate;
+
+    private int serviceId;
+
+    private int actor;
+
+    private int versionNo;
+
+    private PcqAnswers pcqAnswers;
+
+    public PcqAnswerRequest() {
+        // Intentionally left blank.
+    }
+
+    public PcqAnswerRequest(int pcqId) {
+
+        this.pcqId = pcqId;
+    }
+
+    public String getCaseId() {
```

Comment:
We should be using Lombok annotations here it would be worth reading up as it will save a lot of coding. Getters or setters can be produced using annotations like @NoArgsConstructor and then you can use Data annotation to add a builder to instantiate these objects on one line.